gear/requirements.txt

```diff
@@ -14,3 +14,4 @@ prometheus_async>=19.2.0,<20
 prometheus_client>=0.11.0,<1
 PyMySQL>=1,<2
 sortedcontainers>=2.4.0,<3
+orjson>=3.6.4,<4
```

Comment:
This should be inherited from hailtop requirements, why include it again here?

Reply:
Ya I went back and forth for a bit on this but I think I made the wrong choice. `gear` imports `hailtop` so it *needs* all of `hailtop`'s requirements. But instead of using `-r` and having duplicative `pinned-requirements.txt` files for every service, I used `-c` and install a bunch of pinned-requirements.txt files in the Dockerfiles. But I think the better thing to do is

- have gear `-r` the hailtop requirements and `-c` the hail and dev requirements
- have the services `-r` the gear requirements
- in the service dockerfiles now we can just install the service's `pinned-requirements.txt`


I think I didn't want requirements to be duplicated across different pinned-requirements files with different versions, but this approach isn't immune to that problem because of transitive dependencies. And we have a more robust way of ensuring that all the requirements are in sync anyway.